### ISSUE TITLE
move  hass folder on ./store folder

### ISSUE THREAD

**Issue by y1ann1s** [OpenZWave/Zwave2Mqtt#146]
**Is your feature request related to a problem? Please describe.**
devices not translated correctly

**Describe the solution you'd like**
move  hass folder on ./store folder so to be able to add devices without rebuilding image, I understand the entries here are not generic as they include node id 

I want to add Qubino and Horstman devices

**Comment by robertsLando** [comment]
I don't understand well your question but I think you mean you want to move the zwave devices config database folder (the one that contains all zwave devices database) to store folder. 

If so you can already do this by specify in zwave settings a custom db path in `config path` entry.

For example you could create a folder in store named `config` and inside it move all configuration files and set `config path` of zwave settings to `/usr/src/app/store/config`

**Comment by Munsio** [comment]
I think what @y1ann1s describes is that the docker image for hassio does not expose the `hass` folder to the host file system.
Therefore it is not possible to make changes that are persistent.

I opened a issue on the hassio image repo: https://github.com/hassio-addons/addon-zwave2mqtt/issues/11 but it got closed by the bot and i never got any answer so didn't want to put time into a pull request when i don't know if it gets accepted or not.

I think it would also be great if it would be possible to change the config folder as you described and therefore the location of the `devices.js` which is used to identify non generic devices.
